ArchiSteamFarm/Steam/Bot.cs

```diff
@@ -2859,6 +2859,8 @@ public sealed class Bot : IAsyncDisposable {
 					SteamFriends.SetPersonaState(BotConfig.OnlineStatus);
 				}
 
+				ArchiHandler.SetPersonaStateFlags(BotConfig.OnlineFlags);
```

Comment:
I'd guard this in `if (flags > 0)` as we don't need to send a request if no flags are specified, other than that looks really good, feel free to test if it still works for you when you set new config prop to `2048`.

Reply:
![image](https://user-images.githubusercontent.com/51455765/153421865-2a733821-9749-4bf5-931e-01e59a6c2bf7.png)

Reply:
![image](https://user-images.githubusercontent.com/51455765/153422168-cc883989-71d5-4d5a-86c6-4818733ca3e8.png)
Everything's cool.